_includes/posts-list.html

```diff
@@ -4,6 +4,9 @@
     <li>
       <div class="row">
         <div class="col-md-12">
+          {% if post.cover_photo %}
+          <img src="{{ post.cover_photo }}" class="cover-photo">
```

Comment:
Nit: Indent.

Add `alt` attribute. Probably can use the `alt="{{ post.title }} cover photo"`

Reply:
will fix